docs/jobs/state.md

```diff
@@ -148,19 +148,32 @@ The input state looks like this:
 
 ### Cron triggered runs
 
-On the platform, when a Run is triggered by a cron job, the input state will the
-final output state of the **last succesful run** for this workflow. This allows
-each subsequent run to _know_ about previous runs—i.e., you can pass information
-from one run to another even if they happen days apart.
+#### What are cron jobs?
+
+A **cron job** is a time-based scheduler that automatically runs your workflow at specified intervals. Think of it like setting an alarm clock for your workflow. It will run automatically at the times you configure, whether that's every hour, daily at 9 AM, or on the first day of each month.
+
+
+#### How cron input state works
+
+When a run is triggered by a cron job, the input state will be the final output state of the **last successful run** of the first step in the workflow. This allows each subsequent run to _know_ about previous runs. In other words, you can pass information from one run to another even if they happen days apart.
```

Comment:
> the input state will be the final output state of the **last successful run** of the first step in the workflow.

Didn't we agree this was incorrect? It's the state of the  cron step that goes into the next workflow - not the FINAL step.

See [slack thread](https://openfn.slack.com/archives/C0394D0GBUN/p1760378160109549) and pick up the conversation there if you're not sure. You should be able to easily test the actual behaviour against the app.